### DISCUSSION TITLE
DCB file question

### DISCUSSION THREAD

**Discussion by SquirrelKnight** [tomojitakasu/RTKLIB#738] [Q&A]
Hello everyone. I am working with PPP-K data, which I need to be quite accurate with. I understand the DCB files are very important for post-processing GNSS data. The time series that I have is long-running, taking place over several months. I have a few fairly straightforward questions that I hope can be answered:

1. Can RTKlib handle multiple DCB files, or does using a wildcard when specifying files (e.g. '*.DCB') lead to one file overriding the others?
2. Does RTKlib have support for the CODE DCB files provided on their website?: http://ftp.aiub.unibe.ch/CODE_MGEX/CODE/2022/
3. Does RTKlib have support for SINEX Bias files (BSX)? It seems this the way a lot of organizations are moving towards to accompany MGEX data.

Any answers/responses are greatly appreciated!

**Comment by andrea-raiola-kntn** [comment]
Hi you managed to know if the BIA support is present?

**Reply by SergeyDolin** [reply]
You can try my solution https://github.com/tomojitakasu/RTKLIB/pull/737
